vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/src/test/java/com/vaadin/flow/component/grid/dataview/ItemCountEstimateGridIT.java

```diff
@@ -99,10 +99,10 @@ public void itemCountEstimateGrid_estimateLessThanCurrentRange_estimateNotChange
         grid.scrollToRow(1500);
         setEstimate(1501);
 
-        verifyRows(1501);
+        verifyRows(1800);
 
-        setEstimate(1600);
-        verifyRows(1600);
+        setEstimate(1700);
+        verifyRows(1700);
```

Comment:
The previous tests were written with the assumption that the scrollToIndex wouldn't continue to the desired index once the last scrollable index before increasing the range has been reached. This works as expected with the new version so the grid actually ends up at index 1500 after `scrollToRow(1500)`, even if the effective size is increased dynamically.